packages/grafana-ui/src/components/GraphNG/GraphNG.tsx

```diff
@@ -110,7 +122,7 @@ export const GraphNG: React.FC<GraphNGProps> = ({
   const configRev = useRevision(frame, compareFrames);
 
   const configBuilder = useMemo(() => {
-    const builder = new UPlotConfigBuilder();
+    const builder = new UPlotConfigBuilder(stacking.enable);
```

Comment:
Not sure the best approach, maybe builder.enableStacking() would align better with the builder's api?